teuthology/task/install.py

```diff
@@ -926,6 +925,104 @@ def _upgrade_deb_packages(ctx, config, remote, debs):
         ] + ['%s=%s' % (d, version) for d in debs],
     )
 
+@contextlib.contextmanager
+def rh_install(ctx, config):
+    """
+    Installs rh ceph on all hosts in ctx.
+
+    :param ctx: the argparse.Namespace object
+    :param config: the config dict 
+    """
+    with parallel() as p:
+        for remote in ctx.cluster.remotes.iterkeys():
+            if remote.os.name == 'rhel':
+                log.info("Installing on RHEL node")
+                p.spawn(rh_install_pkgs, ctx, remote)
+            else:
+                raise RuntimeError ( "Test requires RHEL nodes")
+    try:
+       yield
+    finally:
+        if config.get('skip_uninstall'):
+           log.info("Skipping uninstall of Ceph")
+        else:
+           rh_uninstall(ctx=ctx, config=config)
+
+           
+def rh_uninstall(ctx, config):
+    """
+    Uninstalls rh ceph on all hosts.
+
+    :param ctx: the argparse.Namespace object
+    :param config: the config dict
+    """
+    with parallel() as p:
+        for remote in ctx.cluster.remotes.iterkeys():
+            p.spawn(rh_uninstall_pkgs, ctx, remote)
+    
+
+def rh_install_pkgs(ctx, remote):
+    """
+    Installs RH build using ceph-deploy.
+
+    :param ctx: the argparse.Namespace object
+    :param remote: the teuthology.orchestra.remote.Remote object
```

Comment:
Successful install should be sufficient, since the ansible task will set the version to right repo, It all depends on what repo is being set on the nodes.